src/test/fuzz/key.cpp

```diff
@@ -305,4 +306,25 @@ FUZZ_TARGET_INIT(key, initialize_key)
             assert(key == loaded_key);
         }
     }
+
+    {
+        std::array<uint8_t, 32> rnd32;
+        memcpy(rnd32.data(), &random_uint256, 32);
+        assert(pubkey.EllSqEncode(rnd32)->size() == ELLSQ_ENCODED_SIZE);
+    }
+}
+
+FUZZ_TARGET_INIT(ellsq, initialize_key)
+{
+    if(buffer.size() < 64) {
```

Comment:
You may install `clang-format` and run the https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy script to preempt whitespace nitpicking.

Reply:
Done.